docs/dashboards-new/link-dashboards.md

```diff
@@ -22,15 +23,14 @@ When you click a data point on a Dashboard (New) chart the Summary tab provides
 * The following image shows a Summary panel from a selected log data point. The **Linked Dashboards** section is at the bottom.<br/>  ![summary v2 logs.png](/img/dashboards-new/drill-root-causes/summary-v2-logs.png)
 * The following image shows a Summary panel from a selected metric data point. The **Linked Dashboards** section is at the bottom.<br/>  ![metrics summary v2.png](/img/dashboards-new/drill-root-causes/metrics-summary-v2.png)
 
-
 ## Link a Dashboard (New) to Explore
```

Comment:
```suggestion
## Linking a dashboard (new) to explore
```

Reply:
@JainM6 Dashboard (New) and Explore are UI terms, I have updated the heading with more meaningful info.